FirebaseSessions/Sources/FirebaseSessions.swift

```diff
@@ -87,6 +94,7 @@ protocol SessionsProvider {
     super.init()
 
     self.initiator.beginListening {
+      self.settings.fetchAndCacheSettings()
```

Comment:
NIT: Thanks for bringing it in. Can you add a comment to state that we start with updating the remote configurations?

Reply:
Done